temba/channels/types/facebookapp/type.py

```
@@ -76,8 +80,8 @@ def deactivate_trigger(self, trigger):
                 url, json=body, params={"access_token": access_token}, headers={"Content-Type": "application/json"}
             )
 
-            if response.status_code != 200:  # pragma: no cover
-                raise Exception("Unable to update call to action: %s" % response.text)
+            if response.status_code != 200:
+                logger.error("Unable to update call to action: %s" % response.text)
```

This doesn't feel like the right place for this. In the case of `ChannelType.deactivate` it throws an exception and the calling code, e.g. Channel.release decides whether to ignore it.

Seems like for consistency we should let the channel type functions be dumb.. `Trigger.archive` can decide to ignore exceptions and log them.

Logging to sentry also doesn't feel quite right here (are we going to investigate these?) but maybe better than just ignoring errors completely.